src/WebJobs.Extensions.DurableTask/Scale/DurableTaskTriggersScaleProvider.cs

```diff
@@ -40,7 +39,16 @@ public DurableTaskTriggersScaleProvider(
             this.GetOptions(triggerMetadata);
 
             IDurabilityProviderFactory durabilityProviderFactory = this.GetDurabilityProviderFactory();
-            DurabilityProvider defaultDurabilityProvider = durabilityProviderFactory.GetDurabilityProvider();
+
+            DurabilityProvider defaultDurabilityProvider;
+            if (durabilityProviderFactory.Name == "azureManaged")
```

Comment:
Is there any way we can avoid checking for a string literal here? Like is "azureManaged" a constant defined somewhere for example?